app/services/dashboard/recent_work_service.rb

```diff
@@ -32,6 +32,7 @@ def call
                       .projects_joins(:from_experiments)
                       .repositories_joins
                       .reports_joins
+                      .where('repositories.archived IS NOT TRUE')
```

Comment:
I think we have same in `Activity.rb:51` Check if doing same thing and remove one.